pyproject.toml

```diff
@@ -13,7 +13,8 @@ markers = [
 [tool.mypy]
 show_column_numbers = true
 show_error_codes = true
-exclude = "build"
+exclude = ["build"]
+ignore_missing_imports = true
```

Comment:
Are we sure we want to ignore missing imports globally? Isn't there a way to ignore specific cases locally instead?